### PR TITLE
chore: Default PR template and added link to PR acceptance checklist

### PR DESCRIPTION
**Pull request by joshsadam**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

- Moves the existing template down into the `.github` directory, and renamed to `pull_request_template.md`
- Added a link the the PR acceptance checklist so that developers could easily review it when making their PR

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

N/A

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Unfortunately cannot validate locally

## PR acceptance checklist
[This checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [X] I have evaluated the PR acceptance checklist for this PR.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ _Screenshots are required for UI changes, and strongly recommended for all other
 _Numbered steps to set up and validate the change are strongly suggested._
 
 ## PR acceptance checklist
-This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
+[This checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
 
 - [ ] I have evaluated the PR acceptance checklist for this PR.
```
Comment (ericenns) [inline .github/pull_request_template.md:13]: **nitpick (blocking):** Can you move the link to the text `PR acceptance checklist` below instead, that way we are matching Gitlab.

**Review by ericenns — CHANGES_REQUESTED** [review]
Looking great, minus link location

File: .github/pull_request_template.md (hunk 1 above)
Comment (joshsadam) [reply .github/pull_request_template.md]: Sounds good to me, updated in a55430909a0fb398fdf04186aa2ca2b11aee6026

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Pull request merged**